### DISCUSSION TITLE
component channel test fail on ubuntu16.04

### DISCUSSION THREAD

**Discussion by truewq** [minhanghuang/CyberRT#10] [Q&A] [answered]
Hi @minhanghuang 
大佬
这个测试也不能正常通信，有什么建议吗？
source setup.bash
cyber_launch start share/examples/common.launch
./cyber/examples/common_component_example/channel_prediction_writer
./cyber/examples/common_component_example/channel_test_writer

![image](https://user-images.githubusercontent.com/7168949/206453164-81817556-cafd-4d8c-b148-078eef9a521c.png)
![image](https://user-images.githubusercontent.com/7168949/206453235-77a394d8-7ab6-4951-a3a8-a039ea5675ff.png)
![image](https://user-images.githubusercontent.com/7168949/206453296-cdb6a97e-8cb9-4879-9d39-3e4aa01d6e43.png)

**Comment by minhanghuang** [comment, marked as answer]
> Hi @minhanghuang 大佬 这个测试也不能正常通信，有什么建议吗？ source setup.bash cyber_launch start share/examples/common.launch ./cyber/examples/common_component_example/channel_prediction_writer ./cyber/examples/common_component_example/channel_test_writer
> 
> ![image](https://user-images.githubusercontent.com/7168949/206453164-81817556-cafd-4d8c-b148-078eef9a521c.png) ![image](https://user-images.githubusercontent.com/7168949/206453235-77a394d8-7ab6-4951-a3a8-a039ea5675ff.png) ![image](https://user-images.githubusercontent.com/7168949/206453296-cdb6a97e-8cb9-4879-9d39-3e4aa01d6e43.png)

需要配置一下日志, 或者用`std::cout`打印
![image](https://user-images.githubusercontent.com/37317982/206635067-d51db05a-fa18-4979-b0d0-9ecb2d6f936a.png)

**Reply by truewq** [reply]
查看AINFO里面发现通信正常的，谢谢大佬

**Reply by minhanghuang** [reply]
> 查看AINFO里面发现通信正常的，谢谢大佬

共勉.
